### ISSUE TITLE
TypeError: Cannot read property 'ember-component-css' of undefined

### ISSUE THREAD

**Issue by MariannaAtPlay** [ember-codemods/tagless-ember-components-codemod#62]
Hello,

trying to use this tool for the first time to upgrade one component.

Command used:
`npx tagless-ember-components-codemod app/components/multiproducts/crt-product-event-card.js`

Output:  
```npx: installed 376 in 10.716s
 🔍  Searching for component files...
TypeError: Cannot read property 'ember-component-css' of undefined
    at runForPath (/Users/mshvarts/.npm/_npx/91269/lib/node_modules/tagless-ember-components-codemod/lib/index.js:27:21)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async run (/Users/mshvarts/.npm/_npx/91269/lib/node_modules/tagless-ember-components-codemod/lib/index.js:12:3)
```

Any guidance to resolve this is appreciated.

Marianna

**Comment by Turbo87** [comment]
this has been fixed by https://github.com/ember-codemods/tagless-ember-components-codemod/pull/53, but it looks like we forgot to release this to npm 😱

**Comment by Turbo87** [comment]
released as v0.4.0 :)
